docs/source/develop.rst

```diff
@@ -288,6 +288,58 @@ will be run each time you commit changes. You can skip these checks with
 ``git commit --no-verify`` or with the short version ``git commit -n``.
 
 
+Dask CI Infrastructure
```

Comment:
Can we move this section to the end of this page? Currently it's between the "Contributing to Code" and "Contributing to Documentation" sections, which seems a little out of place